### PR TITLE
Updates for new version of RPS

### PR DESCRIPTION
**Pull request by TheChronicMonster**
New folder structure and updated markdown for Rock, Paper, Scissors

## Summary

I've copied the current /tut-n/ directory structure with mirrored /rps-n-lorem/ directories in /examples/.
This new structure will allow for more expansive getting started docs without disturbing the scrbl documentation that is currently in use. 

The new RPS index.md tutorial is 90% complete. 
I need to update URLs for the snippets, double check grammar/typos, and add a few subheadings. However, all Primary content and structure is in place.

### CODE REVIEW

**Review by jeapostrophe — CHANGES_REQUESTED** [review]
I think rather than making new `examples/rps-X-Y` directories, we should rename the old ones and update the emeritus site too. (A simple `git mv`. If you want, I can do that and you change just commit the `docs` changes)

**Comment by TheChronicMonster** [comment]
Updated tut-n directories and references to matching rps-n-lorem structure.

Calls to tut-n related files were pervasive throughout the system. References to tut-n files have been replaced so links remain intact. 

Built repo in /docs/ locally and tested current and new developer portals. Code snippets load properly, links work as expected. However, links to source code in GH currently leads to 404, but I believe this will resolve itself once the files and directories are updated.

File: docs/dev/rewire.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ r guide-solidity guide/solidity
 r guide-timeout guide/timeout
 r guide-versions guide/versions
 r guide-windows guide/windows
-r overview tut/overview
+r overview rps/overview
```
Comment (jeapostrophe) [inline docs/dev/rewire.sh:33]: This should still be `tut`

File: docs/dev/rewire.sh [hunk 2]
```diff
@@ -56,8 +56,8 @@ r ref-programs-consensus reach/consensus
 r ref-programs-local reach/local
 r ref-programs-module reach/module
 r ref-programs-step reach/step
-r tut tut/rps
-r tut-7-rpc tut/rps/7-rpc
+r rps rps/rps
+r rps-7-rpc rps/rps/7-rpc
```
Comment (jeapostrophe) [inline docs/dev/rewire.sh:60]: These too

File: docs/scrbl/guide-limits.scrbl [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 @title[#:version reach-vers #:tag "guide-limits"]{What are Reach's limitations?}
 
-Today, Reach is a powerful language for building decentralized applications, as demonstrated in @seclink["overview"]{the overview}, @seclink["tut"]{the tutorial}, and the @seclink["workshop"]{workshop series}.
+Today, Reach is a powerful language for building decentralized applications, as demonstrated in @seclink["overview"]{the overview}, @seclink["rps"]{the tutorial}, and the @seclink["workshop"]{workshop series}.
```
Comment (jeapostrophe) [inline docs/scrbl/guide-limits.scrbl:6]: You can't rename the sections, because that will break the old links.

File: docs/scrbl/guide-rpc.scrbl [hunk 4]
```diff
@@ -15,7 +15,7 @@ Presently, Reach provides RPC client implementations for:
 
 ]
 
-The @seclink["tut-7-rpc"]{tutorial section on RPC-based frontends} provides a walkthrough of using these libraries.
+The @seclink["rps-7-rpc"]{tutorial section on RPC-based frontends} provides a walkthrough of using these libraries.
```
Comment (jeapostrophe) [inline docs/scrbl/guide-rpc.scrbl:18]: Ditto

File: examples/tut-7-array/index.mjs [hunk 5]
```diff
@@ -1,71 +0,0 @@
-import { loadStdlib } from '@reach-sh/stdlib';
```
Comment (jeapostrophe) [inline examples/tut-7-array/index.mjs:1]: You should not have deleted this directory

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

**Pull request closed without merging**